.buildkite/pipeline.yml

```diff
@@ -0,0 +1,29 @@
+steps:
+
+  - command: |-
+      echo "~~~ pip install"
+      echo 'odus' | sudo -S rm -rf /tmp/pip_build_root/*
+      echo 'odus' | sudo -S pip install -r requirements.txt
+
+      echo "~~~ test"
+      python -m unittest discover tests
+
+    label: tests
+    agents:
+      - type=native
+
+  - wait
+
+  - command: |-
+      echo "~~~ pip install"
+      echo 'odus' | sudo -S rm -rf /tmp/pip_build_root/*
+      echo 'odus' | sudo -S pip install -r requirements.txt
+
+      echo "~~~ publishing on pypi"
+
```

Comment:
deployment command missing?

Reply:
This PR was a mistake. It's  just working in progress stuff, i'll close it now.